tests/integration/cli/tests/snapshot.rs

```diff
@@ -1281,3 +1281,57 @@ fn test_snapshot_fs_rename() {
         .run_wasm(include_bytes!("./wasm/fs-rename.wasm"));
     assert_json_snapshot!(snapshot);
 }
+
+#[cfg_attr(any(target_env = "musl", target_os = "windows"), ignore)]
+#[test]
+fn test_snapshot_exit_0_from_main() {
```

Comment:
Yay, tests, amazing.

I think these would have been a better fit for the new Rust-based test library, since that allows for definining the expected exit code I think ... but fine with leaving them here also.

Although will need to be very careful with the outputs here.
If someone else sees changes here it would be easy to accept a new snapshot with changed values.

Can we add an additional assert_eq() for the expected exit code?

Reply:
Snapshots already include the exit code as well, so there's no need for anything more here: https://github.com/wasmerio/wasmer/pull/5153/files#diff-499a0af35079fb7373fe5bc28ca85c9b84fae898427d4bcba948dbf56ebaebdfR19

Reply:
Yeah they do, my point is that it would be easy to accept a snapshot with a different exit code if you don't know the expected exit code. 

If someone else touches the tests in the future.

Reply:
Well, truth be told, snapshot tests are the only thing I know how to make. I do think this is fine (the exit code is mentioned in the test names as well as the source code and the snapshot) but if you think we absolutely need to change it, feel free to point me at the new test library you mentioned.